apps/meta-data-utils/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
```

Comment:
ouch, this is quite some config. How to know it is okay now?

Reply:
there is no 'okay' , just config, could try and share the tsconfig between projects